readme.md

```diff
@@ -29,9 +29,11 @@ Download: [nexusmods.com](https://www.nexusmods.com/pathfinderwrathoftherighteou
 # Development Setup
 1. Install ToyBox mod into your game via Unity Mod Manager
 1. Clone the git repo
-1. Locate the install folder of Pathfinder Wrath of the Righteous
-1. Go to System Properties > Environment Variables and add WrathPath with a value that looks like this:
+1. You can contribute to Rogue Trader, Wrath or both of them. If you only want to contribute to one you can unload the unused one in Visual Studio and only need to setup the variable for the one you need.
```

Comment:
We should also say to take care to not break the other project and to develop against both as soon as possible.